### ISSUE TITLE
Style/RedundantCurrentDirectoryInPath error on v 1.53.0

### ISSUE THREAD

**Issue by matas-zanevicius** [rubocop/rubocop#11974]
On Dependabot branch that aims to upgrade rubocop version.

--------

## Expected behavior

Cops should not cause errors

## Actual behavior

An error occurred while Style/RedundantCurrentDirectoryInPath cop was inspecting /Users/[..]/config.ru:6:2.
undefined method `start_with?' for nil:NilClass
[]../.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/rubocop-1.53.0/lib/rubocop/cop/style/redundant_current_directory_in_path.rb:26:in `on_send'

## Steps to reproduce the problem

Code line inspected when error was raised.
```ruby
require_relative("routes/#{module_name}")
```

## RuboCop version

```console
1.53.0 (using Parser 3.2.2.3, rubocop-ast 1.29.0, running on ruby 3.2.2) [arm64-darwin21]
  - rubocop-graphql 1.3.0
  - rubocop-minitest 0.31.0
  - rubocop-performance 1.18.0
  - rubocop-rake 0.6.0
  - rubocop-sequel 0.3.4
```
